### PR TITLE
Update API docs

### PR DESCRIPTION
**Pull request by zhssh**
Add documentation for missing API endpoints

### CODE REVIEW

File: content/source/docs/enterprise/api/oauth-clients.html.md [hunk 1]
```diff
@@ -25,10 +153,12 @@ This endpoint allows you to create a VCS connection between an organization and
...
 * [GitLab, GitLab Community Edition, and GitLab Enterprise Edition](https://docs.gitlab.com/ce/user/profile/personal_access_tokens.html#creating-a-personal-access-token)
 
-Result  | Status and response
---------|------------------------
-Success | HTTP 201 and a JSON API document (`type: "oauth-clients"`).
-Failure | HTTP 4XX and a JSON API list of error messages.
+Status  | Response                                        | Reason
+--------|-------------------------------------------------|----------
+[201][] | [JSON API document][] (`type: "oauth-clients"`) | OAuth Client successfully created
+[404][] | [JSON API error object][]                       | OAuth Client not found or user unauthorized to perform action
```
Comment (nfagerlund) [inline content/source/docs/enterprise/api/oauth-clients.html.md]: "not found" doesn't seem to make sense for a create request, right? So it's probably only the unauthorized case.

File: content/source/docs/enterprise/api/oauth-tokens.html.md [hunk 2]
```diff
@@ -12,50 +12,218 @@ The `oauth-token` object represents a VCS configuration which includes the OAuth
...
-      },
-      "links": {
-        "self":"/api/v2/oauth-tokens/ot-hmAyP66qk2AMVdbJ"
       }
+    },
+    "links": {
+      "self": "/api/v2/oauth-tokens/ot-hmAyP66qk2AMVdbJ"
     }
-  ]
+  }
```
Comment (nfagerlund) [inline content/source/docs/enterprise/api/oauth-tokens.html.md]: Did the response body change from an array of objects to a single object? Or was that a copy-paste issue?

Comment (zhssh) [reply content/source/docs/enterprise/api/oauth-tokens.html.md]: The response had been copied from another part of the docs. It's been fixed. Thanks!

File: content/source/docs/enterprise/api/oauth-clients.html.md (hunk 1 above)
Comment (zhssh) [reply content/source/docs/enterprise/api/oauth-clients.html.md]: If the organization is not found the response will be 404. Should this be corrected to "Organization"?

Comment (zhssh) [reply content/source/docs/enterprise/api/oauth-clients.html.md]: The label has been updated.

**Pull request merged**